docs/source/index.rst

```diff
@@ -2,23 +2,39 @@
 openwisp-radius
 ===============
 
-.. image:: https://travis-ci.org/openwisp/openwisp-radius.svg
+.. image:: https://travis-ci.org/openwisp/openwisp-radius.svg?branch=master
    :target: https://travis-ci.org/openwisp/openwisp-radius
-   :alt: Automated Build
+   :alt: CI build status
 
-.. image:: https://coveralls.io/repos/openwisp/openwisp-radius/badge.svg
-   :target: https://coveralls.io/r/openwisp/openwisp-radius
-   :alt: Coverage
+.. image:: https://coveralls.io/repos/github/openwisp/openwisp-radius/badge.svg?branch=master
+   :target: https://coveralls.io/github/openwisp/openwisp-radius?branch=master
+   :alt: Test Coverage
 
 .. image:: https://requires.io/github/openwisp/openwisp-radius/requirements.svg?branch=master
    :target: https://requires.io/github/openwisp/openwisp-radius/requirements/?branch=master
    :alt: Requirements Status
 
-.. image:: https://img.shields.io/pypi/v/openwisp-radius
-   :target: https://pypi.org/project/openwisp-radius
-   :alt: PyPI
+.. image:: https://img.shields.io/gitter/room/nwjs/nw.js.svg
+   :target: https://gitter.im/openwisp/general
+   :alt: Chat
 
-openwisp-radius is part of the `OpenWISP project <http://openwisp.org>`_.
+.. image:: https://badge.fury.io/py/openwisp-radius.svg
+   :target: http://badge.fury.io/py/openwisp-radius
+   :alt: Pypi Version
+
+.. image:: https://pepy.tech/badge/openwisp-radius
+   :target: https://pepy.tech/project/openwisp-radius
+   :alt: Downloads
+
+.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
+   :target: https://pypi.org/project/black/
+   :alt: code style: black
+
+.. image:: images/demo_radius.gif
+   :alt: Feature Highlights
+
+**openwisp-radius** is Django reusable app that provides an admin interface to a
+`freeradius <http://freeradius.org/>`_ database.
 
 .. image:: http://netjsonconfig.openwisp.org/en/latest/_images/openwisp.org.svg
```

Comment:
can you switch to this logo please? https://raw.githubusercontent.com/openwisp/openwisp2-docs/master/assets/design/openwisp-logo-black.svg